### ISSUE TITLE
Add AJAX functionality

### ISSUE THREAD

**Issue by joecridge** [danielswaine/SodburyChallenge#59]
- Periodically update the clock with server time on all pages
- Update finish times on public home page
- Update all times on private home page

**Comment by danielswaine** [comment]
Having a quick google, there's a gem called Best in Place, could this be used to update the required times?

**Comment by joecridge** [comment]
Yes that would be great! Having a separate form for them was going to be a bit cumbersome. Do you fancy giving it a go?

**Comment by danielswaine** [comment]
I'll give it a go :) it looks reasonably simple, hope I haven't spoken too soon!

**Comment by danielswaine** [comment]
Although looks like you've already started!

**Comment by joecridge** [comment]
All I've done is install it haha, I'll leave it and work on the scoring instead. I think you'll need to replace the if statements cause they only update on refresh.

**Comment by joecridge** [comment]
Could you add it to the challenges page as well? That way we can dispense with the edit button and replace it with a manage route button

**Comment by danielswaine** [comment]
Okay I'll have a go at it. How's the scoring going?

I'll see what I can do...

**Comment by joecridge** [comment]
AJAX instead of `<meta>` refreshing will let us use turbolinks again
